data-plane/dispatcher/src/test/java/dev/knative/eventing/kafka/broker/dispatcher/RecordDispatcherTest.java

```diff
@@ -57,7 +56,7 @@ public void shouldNotSendToSubscriberNorToDeadLetterSinkIfValueDoesntMatch() {
     );
 
     final var record = record();
-    dispatcherHandler.handle(record);
```

Comment:
The `handle` invokes the `apply`, which does the actual execution chain.   Now, we will call `dispatch`.

Out of curiousity, could this test have also call directly `apply`, rather than the wrapper?

Reply:
There is not `apply` method anymore in `RecordDispatcher`, but yes using the previous shape of the file, it could have

Reply:
yeah, I meant about the past version.